material/30days-curriculum/day08_サイドバーを完成させよう.md

```
@@ -625,19 +650,23 @@ npm run dev
+![ログイン画面。メールアドレスとパスワードの入力欄、ログインボタン、登録リンクが並んでいる](./screenshots/day05/login.png)
 
 **確認フロー**:
 
 1. `/dashboard` へアクセス → middleware が `/login` にリダイレクト。
 2. `admin@example.com` / `password123` でログイン。
 3. ダッシュボードが表示される（サイドバー付き）。
 
-![ダッシュボードとサイドバー](./screenshots/dashboard.png)
+![左に3項目のサイドバー、右に Day 02 で作ったダッシュボードのカードが並んでいる画面](./screenshots/day08/dashboard.png)
+
 4. サイドバーに「ダッシュボード」「プロジェクト」「マイタスク」の3つのメニューが見える。
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)